engine/src/main/java/com/redhat/cloud/notifications/recipients/itservice/pojo/request/ITUserRequest.java

```diff
@@ -0,0 +1,45 @@
+package com.redhat.cloud.notifications.recipients.itservice.pojo.request;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.LinkedList;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ITUserRequest {
+
+    public ITUserRequestBy by;
+    public Include include;
+
+    public ITUserRequest(String accountId, boolean adminsOnly) {
```

Comment:
As I commented in the previous PR, many things in this constructor should be `static`. We shouldn't rebuild all objects for each request.